package.json

```diff
@@ -31,5 +28,11 @@
   },
   "dependencies": {
     "handlebars": "^4.0.6"
+  },
+  "externals": {
```

Comment:
Check the getInfo() Api of the module, that will expose an array. 
This allows for some added flexibility, for example for the React and ReactDOM externals, I'm using this configuration to also define the global names for the 'externals' within the webpackconfigurator, so that I'm sure what is exposed in getInfo is insync with how we bundle and setup those externals => https://github.com/opentable/oc-template-react/pull/1/files#diff-ce59cc337ce2a5235282caeb524de333R16